sql/grant_tables/role_edge.go

```diff
@@ -72,3 +75,25 @@ func (r *RoleEdge) Equals(ctx *sql.Context, otherEntry in_mem_table.Entry) bool
 	}
 	return *r == *otherRoleEdge
 }
+
+// ToStringWithQuote returns the "TO" user as a formatted string using the quotes given. If a replacement is given, then
+// also replaces any existing instances of the quotes with the replacement.
+func (r *RoleEdge) ToStringWithQuote(quote string, replacement string) string {
```

Comment:
This method appears to be unused

Reply:
Makes sense to me that if the From users have a way to stringify them, then the To users have a matching method. Especially when these are gonna be given to integrators.